includes/Stories_Renderer/Renderer.php

```diff
@@ -491,7 +532,7 @@ protected function render_story_with_story_player() {
 
 		?>
 		<amp-story-player style="<?php echo esc_attr( $player_style ); ?>"
-			<?php echo( esc_attr( $story_player_attributes ) ); ?>>
+			<?php echo( $story_player_attributes ); // phpcs:ignore -- Outputs escaped string of attributes. ?>>
```

Comment:
Instead of the comment, you define the rule you are ignoring. Like [this](
https://github.com/google/web-stories-wp/blob/31dffad61c908082a721218f428b7a88ea0ea285/includes/templates/frontend/single-web-story.php#L38).

Reply:
+1 to be more specific here with `// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped`